igvm/hypervisor.py

```diff
@@ -869,6 +869,43 @@ def hv_cpu_util_overall(self, vm: VM) -> float:
 
         hv_cpu_util_dest = self.hv_predict_vm_cpu_util(vm)
         hv_cpu_util_pct_dest = self.dataset_obj['cpu_util_pct']
-        hv_cpu_util_overall = hv_cpu_util_pct_dest + hv_cpu_util_dest
+        hv_cpu_util_log = self.hv_evaluate_migration_log()
+        hv_cpu_util_overall = hv_cpu_util_pct_dest + \
+                              hv_cpu_util_dest + hv_cpu_util_log
 
         return float(hv_cpu_util_overall)
+
+    def hv_evaluate_migration_log(self) -> int:
+        """HV evaluate migration log
+
+        Calculate the added cpu usage from previous migrations
+
+        :param: none
+        :return: added cpu usage from migrations as integer
+        """
+
+        hv_migration_log = self.dataset_obj['igvm_migration_log']
+        added_cpu_usage = 0
+        for entry in hv_migration_log:
+            cpu_usage_log = entry.split(' ')[1]
+            added_cpu_usage = int(added_cpu_usage) + int(cpu_usage_log)
```

Comment:
No need for a change here but `int(added_cpu_usage)` is not necessary because added_cpu_sage was initialised as integer and only gets summed up with integers so no need for an explicit cast here. For `cpu_usage_log` it must remain.